.travis.yml

```diff
@@ -1,16 +1,39 @@
 sudo: required
 language: ruby
 services:
-  - docker
+- docker
 rvm:
-  - 2.3
-  - 2.4
+- 2.3
+- 2.4
 install:
-  - bundle install --binstubs --path vendor/bundle
+- bundle install --binstubs --path vendor/bundle
 script:
-  - bundle exec rake integration
+- bundle exec rake integration
+before_deploy:
+- bundle exec github_changelog_generator --token $GH_TOKEN
+- git config --global user.email "slapilabs@gmail.com"
+- git config --global user.name "slapibot"
+- export $SLAPI_VERSION=$(bundle exec gem list slapi | grep -o '[0-9]\.[0-9]\.[0-9]')
+- export GIT_TAG=$SLAPI_VERSION.b$TRAVIS_BUILD_NUMBER
+- git tag $GIT_TAG -a -m "SLAPI Build Bot: Generated tag from TravisCI for build $TRAVIS_BUILD_NUMBER"
+- git push -q https://slapibot:$GH_TOKEN@github.com/imperiallabs/slapi --tags
+- ls -R
+deploy:
+  provider: releases
+  prerelease: true
+  file:
+    - Dockerfile
```

Comment:
I just moved it all to the right, I didn't realize the it took the indents away when I re-setup travis.